tests/testsuite/update.rs

```diff
@@ -2693,14 +2648,10 @@ fn update_precise_breaking_incompatible_build_metadata() {
     p.cargo("generate-lockfile").run();
     p.cargo("update -Zunstable-options incompatible --precise 2.0.0+b")
```

Comment:
Does `cargo update --precise <no-meta>` work on packages with metadata?  Whichever way, this should do the same if its a breaking change